cal/engineeringevents.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from datetime import tzinfo, timedelta, datetime, datetime, date, time
+
+from cal.schema import db, User, Event
+from bs4 import BeautifulSoup
+import requests
+
+import re
+
+from icalendar import Calendar
+from datetime import datetime
+
+
+r = requests.get('http://engineering.columbia.edu/feeds/events')
+text = r.text
+soup = BeautifulSoup(text)
+
+def update_engineering_events():
+	events = soup.find_all('item')
+	user = User.query.filter(User.name=="Columbia Engineering").first()
+	for event in events:
+		current_soup = BeautifulSoup(event.renderContents())
+
+		event_name = current_soup.find('title').text
+		url = current_soup.find('link').get_text() #event url
+		url = url.encode('ascii','ignore')
+		event_id = url[len(url) - 5: len(url)]
```

Comment:
`event_id = url[-5:]` works.

Are ids guaranteed to be 5 characters long? Seems brittle to me.